kegeland-frontend/src/components/DataTable.tsx

```diff
@@ -54,6 +55,30 @@ const DataTable = <T extends object>({ data, columns }: DataTableProps<T>) => {
     debugHeaders: true,
     debugColumns: false,
   });
+
+  const navigate = useNavigate();
+
+  {/*]

```

Comment:
Is the commented section to be used later? Or could it be removed?

Reply:
Just an alternative, will remove